crates/dslab-mp/src/system.rs

```diff
@@ -49,6 +49,10 @@ impl System {
         }
     }
 
+    pub fn trace(&self) -> Vec<LogEntry> {
+        self.logger.borrow().trace()
+    }
```

Comment:
Давайте дадим доступ сразу до логгера, может потом пригодится, и заодно это поможет получить трейс без клонирования в логгере:

```suggestion
    pub fn logger(&self) -> RefMut<Logger> {
        self.logger.borrow_mut()
    }
```